### PR TITLE
feat: add social urls and treasury address to space profile

### PR DESCRIPTION
**Pull request by Sekhmet**
## Summary

Closes: https://github.com/snapshot-labs/sx-ui/issues/409
Closes: https://github.com/snapshot-labs/sx-ui/issues/413

This PR adds social urls and treasury address to space profile editor.

**Currently treasury address in execution/treasury page is still hardcoded as we don't support Starknet there. Can be solved in other PR.**

## Test plan

- Use local deployment from https://github.com/snapshot-labs/sx-api/pull/173 or wait for it to be merged.
- Create space/edit space.
- Overview shows correct profile icons.

### CODE REVIEW

**Comment by Sekhmet** [comment]
Will draft for now, will add full flow (with displaying actual values) here, will be mergable once sx-api and sx-subgraphs PRs are merged.

File: src/helpers/__tests__/utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+describe('utils', () => {
+  describe('createErc1155Metadata', () => {
+    it('should create metadata', () => {
+      const metadata = createErc1155Metadata({
+        name: 'Test',
+        description: 'Test description',
+        externalUrl: 'https://test.com',
+        githubUrl: 'https://github.com',
+        twitterUrl: 'https://twitter.com',
+        discordUrl: 'https://discord.com',
```
Comment (bonustrack) [inline src/helpers/__tests__/utils.test.ts:14]: We should not use URLs but usernames instead
```suggestion
        github: 'snapshot-labs',
        twitter: 'SnapshotLabs',
        discord: 'snapshot',
```

File: src/helpers/__tests__/utils.test.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+      expect(metadata).toEqual({
+        name: 'Test',
+        description: 'Test description',
+        external_url: 'https://test.com',
+        properties: {
+          github_url: 'https://github.com',
+          twitter_url: 'https://twitter.com',
+          discord_url: 'https://discord.com',
+          treasury_address: '0x000000000000000000000000000000000000dead'
```
Comment (bonustrack) [inline src/helpers/__tests__/utils.test.ts:26]: This should be an array allowing multiple wallets and we should add the network too, I would name it `wallets` like:
```suggestion
          wallets: [
            'gor:0x000000000000000000000000000000000000dead'
          ]
```
On the UI we don't need to allow input multiple wallets we can force looking at the first wallet.

File: src/components/Block/SpaceFormProfile.vue [hunk 3]
```diff
@@ -14,36 +20,61 @@ const definition = {
...
-    website: {
+    externalUrl: {
       type: 'string',
       title: 'Website',
-      examples: ['Space website URL']
+      examples: ['Website URL']
+    },
+    githubUrl: {
+      type: 'string',
+      title: 'Github',
```
Comment (bonustrack) [inline src/components/Block/SpaceFormProfile.vue:44]: Should be written with the official casing
```suggestion
      title: 'GitHub',
```

File: src/components/Block/SpaceFormProfile.vue [hunk 4]
```diff
@@ -14,36 +20,61 @@ const definition = {
...
+      type: 'string',
+      title: 'Twitter',
+      examples: ['Twitter URL']
+    },
+    discordUrl: {
+      type: 'string',
+      title: 'Discord',
+      examples: ['Discord URL']
+    },
+    treasuryAddress: {
```
Comment (bonustrack) [inline src/components/Block/SpaceFormProfile.vue:57]: Treasury address input should have the contact icon.

File: src/components/Ui/Modal.vue [hunk 5]
```diff
@@ -102,7 +102,6 @@ watch(open, (val, prev) => {
     }
 
     .modal-body {
-      max-height: 420px;
```
Comment (bonustrack) [inline src/components/Ui/Modal.vue:105]: Can you keep the max-height?

**Review by bonustrack — APPROVED** [review]
tACK

**Pull request merged**